src/Components/Aspire.Azure.Storage.Blobs/ConfigurationSchema.json

```diff
@@ -75,15 +90,15 @@
                         "GeoRedundantSecondaryUri": {
                           "type": "string",
                           "format": "uri",
-                          "description": "Gets or sets the secondary storage 'System.Uri' that can be read from for the storage account if the\naccount is enabled for RA-GRS.\n\nIf this property is set, the secondary Uri will be used for GET or HEAD requests during retries.\nIf the status of the response from the secondary Uri is a 404, then subsequent retries for\nthe request will not use the secondary Uri again, as this indicates that the resource\nmay not have propagated there yet. Otherwise, subsequent retries will alternate back and forth\nbetween primary and secondary Uri."
```

Comment:
We left these newlines in purposefully so when they are in the docs, they are in the UI as well.

Today:
![image](https://github.com/dotnet/aspire/assets/8291187/6ffecbcf-81a7-4195-91ad-94a189696957)

After this change:

![image](https://github.com/dotnet/aspire/assets/8291187/c47a8f51-5ed4-4d45-906e-4ba736f65525)

This makes it less readable.

cc @sebastienros

Reply:
That's interesting. The _duplicate_ line break at https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Blobs/src/BlobClientOptions.cs#L171-L180 shows up at https://learn.microsoft.com/en-us/dotnet/api/azure.storage.blobs.blobclientoptions.georedundantsecondaryuri?view=azure-dotnet#azure-storage-blobs-blobclientoptions-georedundantsecondaryuri, but the _singular_ line breaks do not. Are you okay with preserving only the duplicate line breaks?